lib/Basics/VelocyPackHelper.cpp

```diff
@@ -1130,6 +1130,25 @@ uint64_t VelocyPackHelper::extractIdValue(VPackSlice const& slice) {
   return 0;
 }
 
+VPackBuilder VelocyPackHelper::copyObjectWithout(
```

Comment:
looks a bit like VPackCollection::remove(). Why reimplement this here?

Reply:
Removed